### PR TITLE
Footer updates to create parity with Homesite

### PR DESCRIPTION
**Pull request by forrestglick**
Updated links and labels based on the current version of Homesite local footer.

# READY FOR REVIEW

# Summary
- Updates to links in footer to create parity with Homesite offering.

# Review By
-  Nov 15, 2021, if possible

# Criticality
- Not critical.
- Affects all SAA D9 sites.

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Look at Fat Footer
3. Verify links in the footer. Styling and links should match local footer on alumni site (Homesite) - Mixed case added to section labels. Added a link to Access SAA Member Card in lefthand items.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Affects footer across all alumni D9 sites.

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by rebeccahongsf — APPROVED** [review]
LGTM! 😃 any additional comments or work needed @buttonwillowsix ?

File: templates/components/fat-footer/saa-fat-footer.html.twig [hunk 1]
```diff
@@ -109,61 +110,62 @@
...
-    <h2 class="su-local-footer__list-heading">Reading & resources</h2>
+    <h2 class="su-local-footer__list-heading">Reading & Resources</h2>
     <ul class="saa-local-footer__links-column-3">
 {#      <li><a href="/">News/Announcements</a></li>#}
-      <li><a href="https://alumni.stanford.edu/resources/learn/">Webcasts & podcasts</a></li>
+      <li><a href="https://alumni.stanford.edu/stories/">News & Stories</a></li>
+      <li><a href="https://alumni.stanford.edu/resources/learn/">Learn</a></li>
       <li><a href="https://stanfordmag.org/">STANFORD magazine</a></li>
-      <li><a href="https://careers.alumni.stanford.edu">Career connections</a></li>
+      <li><a href="https://careers.alumni.stanford.edu/resources/">Career Connections</a></li>
```
Comment (buttonwillowsix) [inline templates/components/fat-footer/saa-fat-footer.html.twig:135]: @forrestglick we don't want them to go to the main page?

File: templates/components/fat-footer/saa-fat-footer.html.twig [hunk 2]
```diff
@@ -109,61 +110,62 @@
...
-      <li><a href="https://alumni.stanford.edu/perks/">Alumni perks</a></li>
-      <li><a href="https://alumni.stanford.edu/membership/">Alumni membership</a></li>
-      <li><a href="https://sierracamp.alumni.stanford.edu/">Sierra programs</a></li>
+      <li><a href="https://alumni.stanford.edu/perks/">Alumni Perks</a></li>
+      <li><a href="https://alumni.stanford.edu/membership/">Alumni Membership</a></li>
+      <li><a href="https://sierracamp.alumni.stanford.edu/">Sierra Programs</a></li>
       <li><a href="https://cardinalalumni.stanford.edu/get/page/travel-study/overview">Travel/Study</a></li>
-      <li><a href="https://partners.vinoshipper.com/stanford">Wine collection</a></li>
+      <li><a href="https://partners.vinoshipper.com/stanford">Wine Collection</a></li>
       <li><a href="https://cardinalalumni.stanford.edu/get/page/resources/shop/home">Alumni store</a></li>
```
Comment (buttonwillowsix) [inline templates/components/fat-footer/saa-fat-footer.html.twig:148]: If we are capitalizing, shouldn't this go to?

**Review by buttonwillowsix — APPROVED** [review]
I had some comments, but I think this is ready to go. I guess those questions are also ones I would have for the general footer. Approve!

**Comment by buttonwillowsix** [comment]
@rebeccahongsf and @forrestglick I had a couple of questions, but nothing that should stop squashing and merging. I think we are gtg.  @rebeccahongsf do you want to do the honors?

**Comment by rebeccahongsf** [comment]
Thanks @buttonwillowsix ! 😸 I shall go ahead with the squash and merge

**Pull request merged**